### PR TITLE
update API and CLI docs to KES 2023-10-27T22-05-35Z

### PR DESCRIPTION
**Pull request by aead**
This commit updates the server API and CLI docs to reflect the server API and CLI commands provided by the KES server 2023-10-27T22-05-35Z.

It drops various non-useful APIs, like modifying identites and polices, and non-existing CLI commands.

### CODE REVIEW

File: content/cli/_index.md
Comment (ravindk89) [inline content/cli/_index.md:82]: @aead is it worth maintaining something like a deprecated API list for users on older KES versions?

We do something similar for the Server/MC docs where we keep deprecated commands + the version we dropped them from. It slightly simplifies support for users who are pinned to the older version.

cc @djwfyi

Comment (ravindk89) [reply content/cli/_index.md:82]: At a glance, we are dropping:

`enclave`
`manage`
`init`

File: content/cli/_index.md
Comment (djwfyi) [inline content/cli/_index.md]:
```suggestion
Download the binary using `curl`.
Replace `<OS>` and `<ARCH>` with your operating system and CPU architecture.
```

File: content/concepts/server-api.md
Comment (djwfyi) [inline content/concepts/server-api.md]: This is just a writing style thing of using one sentence per line.
It's a technique called "ventilated prose".

```suggestion
The KES server readiness probe. 
Returns `200 OK` if KES is ready.
```

**Review by djwfyi — CHANGES_REQUESTED** [review]
Just two suggestions related to style.
Otherwise, I really appreciate you putting this one in for us!

**Review by ravindk89 — APPROVED** [review]
LGTM!

**Review by djwfyi — APPROVED** [review]
LGTM after I merged my own suggestions.

**Pull request merged**
